extensions/podman/packages/extension/src/utils/notification-manager.spec.ts

```diff
@@ -0,0 +1,282 @@
+/**********************************************************************
+ * Copyright (C) 2023 - 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
+import type { Configuration, Disposable } from '@podman-desktop/api';
+import * as extensionApi from '@podman-desktop/api';
+import { beforeEach, describe, expect, test, vi } from 'vitest';
+
+import * as compatibilityModeLib from './compatibility-mode';
+import { NotificationManager } from './notification-manager';
+import { isDisguisedPodman } from './warnings';
+
+const config: Configuration = {
+  get: () => {
+    // not implemented
+  },
```

Comment:
suggestion: if this is not a copy paste from an existing, you can probably use vi.fn()


```suggestion
  get: vi.fn(),
```